### ISSUE TITLE
Out of date

### ISSUE THREAD

**Issue by slikts** [verekia/js-stack-from-scratch#262]
I've [compiled a list](https://tooling.js.org/) of the currently relevant JS and TS tooling (that I could find), and I've also linked to this resource, since it's very close in focus, but with an added note that it's out of date.

In particular, some things that have changed are:
- Immer for Redux instead of Immutable.js
- Redux Toolkit was introduced
- Emotion and styled-components over JSS
- Increasing focus on API generators like Postgraphile, Hasura, over implementing APIs with Express
- Increasing focus on zero-configuration starters like CRA
- Flow increasingly less relevant compared to TypeScript
- Prettier is very popular

**Comment by verekia** [comment]
You're right, this tutorial is out of date. And your website is fantastic, great job.

I absolutely agree with everything you've listed and learned a few things (like Hot Loader being deprecated), so thank you.

My personal current stack is Gatsby with TypeScript, Apollo, and Emotion, to name only the core libraries.

I don't know what I'm gonna do with this repository to be honest.
